src/NClient.Tools/NClient.DotNetTool/NClient.DotNetTool.csproj

```diff
@@ -0,0 +1,47 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <PackageId>$(AssemblyName)</PackageId>
+        <VersionPrefix>$(VersionPrefix)</VersionPrefix>
+        <VersionSuffix>$(VersionSuffix)</VersionSuffix>
+        <PackageIcon>logo.png</PackageIcon>
+        <PackageProjectUrl>https://nclient.github.io</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/nclient/nclient</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <PackageTags>NClient;CodeGeneration;dotnet-tool</PackageTags>
+        <Description>DotNet CLI tool for generate interfaces of NClient</Description>
+        <Authors>kingmidas74</Authors>
+        <PackageLicenseFile>LICENSE</PackageLicenseFile>
+        <IsPackable>true</IsPackable>
```

Comment:
Instead of this line should be:
```
<PackAsTool>true</PackAsTool>
<ToolCommandName>nclient</ToolCommandName>
```
They used to be, maybe I accidentally deleted them, sorry 😅

Reply:
No problem